docs-source/content/samples/simple/domains/model-in-image/update3.md

```diff
@@ -414,13 +414,17 @@ Here are the steps for this use case:
     Found 1 local data source:
       Datasource 'mynewdatasource': State='Running'
 
+   Found min threads constraint runtime named 'SampleMinThreads' with configured count: 1
+   
+   Found max threads constraint runtime named 'SampleMaxThreads' with configured count: 10
+
```

Comment:
The alignment looks off - the new added output looks like it has a different indent than the original output. Does this look fine in hugo?

Reply:
Thanks for catching this. Fixed.